cli/wallet/wallet.go

```diff
@@ -651,7 +651,8 @@ func importDeployed(ctx *cli.Context) error {
 		return cli.NewExitError("contract has no `verify` method with boolean return", 1)
 	}
 	acc.Address = address.Uint160ToString(cs.Hash)
-	acc.Contract.Script = cs.NEF.Script
+	// Otherwise an old value of some real signature account's script will be used.
```

Comment:
"Explicitly overwrite single signature script of the provided WIF since the contract is known to be deployed."